docs/BasicCommunication/OnAppRegistered/index.md

```diff
@@ -35,30 +35,26 @@ If the application resumes data successfully:
     * `SubscribeButton`
     * `SubscibeVehicleData`
 
-If the application does not resume data successfully:
+If the application does NOT resume data successfully:
 
   * SDL will provide `OnAppRegistered` with `resumeVrGrammars`:`false` or no resume parameter at all.
   * SDL cleans up all previously stored application data for the application that failed to resume. The HMI must also clean up previously compiled `VRGrammars` for the application.
   * The application will send new data to start SDL operations. In this event, SDL and the HMI should restart the cycle of collecting application data for resumption.
 
-!!! MUST
+_**HMI must:**_
```

Comment:
This entire PR (as well as #49, #50, and #51) looks like it was modified to fit the GitHub preview, but these changes do not format as well (compared to the original) on the actual documentation site.

For example: [Original OnSystemRequest page](https://smartdevicelink.com/en/docs/hmi/master/basiccommunication/onsystemrequest/) vs [The OnSystemRequest page for this PR](https://smartdevicelink.com/en/docs/pull_request/f1f00a3259aa9d8d21e0a221f913843f2575690e/basiccommunication/onsystemrequest/)

This files should be formatted with the [SDL Markdown Spec](https://github.com/smartdevicelink/sdl_markdown_spec) in mind. I will point out an example of each of the formatting issue individually with an example.